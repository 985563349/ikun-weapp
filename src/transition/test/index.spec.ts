import path from 'node:path';
import simulate from 'miniprogram-simulate';

describe('transition', () => {
  const KTransition = simulate.load(path.resolve(__dirname, '../transition'), 'k-transition', {
    rootPath: path.resolve(__dirname, '../../'),
  });

  test('should render transition add match snapshot', () => {
    const comp = simulate.render(KTransition);
    comp.attach(document.createElement('parent-wrapper'));

    expect(comp.toJSON()).toMatchSnapshot();
  });
});
