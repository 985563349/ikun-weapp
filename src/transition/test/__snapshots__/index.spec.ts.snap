// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transition should render transition add match snapshot 1`] = `
<wx-view
  class="k-class k-transition "
  id=""
  style="display: none"
  bind:transitionend="onTransitionEnd"
/>
`;
